game/scripts/npc/abilities/silencer_global_silence.txt

```diff
@@ -24,7 +24,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "130 120 110 100 90" //OAA
+    "AbilityCooldown"                                     "100" //OAA
```

Comment:
Last level was meant to be 75 seconds at one point. Use this information however you want.